CareKitStore/CareKitStoreTests/OCKHealthKitStore/TestHealthKitPassthroughStore+Events.swift

```diff
@@ -37,7 +37,7 @@ import XCTest
 // Note, we test the event stream and not the outcome stream because the outcome stream
 // calls into the event stream. Testing the outcome stream is unnecessary.
 
-
+@available(iOS 15, watchOS 8, *)
```

Comment:
If I remember correctly, the majority of devices that run on 15 should also run on 13. I need to look up the limitations for watchOS. So you want the lowest phone and watch to be 15 and 8 respectively?

Reply:
The other issue that will occur with not having `@available(iOS 15, watchOS 8, *)` is the specification of of this line `Package.swift`: https://github.com/carekit-apple/CareKit/blob/449fd35e97b54974895bb3a7741f8294699d9cd3/Package.swift#L7 

Basically, if a developer opens the project using the Package file, they can't run tests because the SPM file specifies a lower target than the Xcode Project